CHANGELOG.rst

```diff
@@ -1,6 +1,22 @@
 Changelog
 =========
 
+0.0.1.dev3 (2018-06-14)
+-------------------------
+Added
+~~~~~
+* Add IRunnable, IMessageHandler
```

Comment:
Mind making this ..gah what's the term for it, basically:

```
``IRunnable``
```

so it's formatted?

Also, mind adding periods at the of these sentences for consistency?